composer.json

```diff
@@ -23,37 +23,45 @@
         "branch-alias": {
             "dev-master": "1.2-dev",
             "dev-develop": "1.3-dev"
+        },
+        "zf": {
+            "module": "ZF\\ContentValidation"
         }
     },
     "require": {
-        "php": ">=5.5",
-        "zendframework/zend-eventmanager": "~2.3",
-        "zendframework/zend-filter": "~2.3",
-        "zendframework/zend-http": "~2.3",
-        "zendframework/zend-inputfilter": "~2.4",
-        "zendframework/zend-mvc": "~2.3",
-        "zendframework/zend-servicemanager": "~2.3",
-        "zendframework/zend-stdlib": "~2.3",
-        "zendframework/zend-validator": "~2.3",
-        "zfcampus/zf-api-problem": "~1.0",
-        "zfcampus/zf-content-negotiation": "~1.0"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-eventmanager": "^2.6.3 || ^3.0.1",
+        "zendframework/zend-filter": "^2.7.1",
+        "zendframework/zend-http": "^2.5.4",
+        "zendframework/zend-inputfilter": "^2.7.2",
+        "zendframework/zend-mvc": "^2.7.9 || ^3.0.2",
+        "zendframework/zend-servicemanager": "^2.7.6 || ^3.1",
+        "zendframework/zend-stdlib": "^2.7.7 || ^3.0.1",
+        "zendframework/zend-validator": "^2.8.1",
+        "zfcampus/zf-api-problem": "^1.2.1",
+        "zfcampus/zf-content-negotiation": "^1.2.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.7",
-        "squizlabs/php_codesniffer": "^2.3.1",
-        "zendframework/zend-loader": "~2.3"
+        "phpunit/phpunit": "^4.8 || ^5.0",
```

Comment:
@weierophinney what is the point to support two version of PHPunit in require-dev?
THe package already has minimum PHP 5.6 as minimum requirement so it's no point.